### PR TITLE
minor language fixes

### PR DESCRIPTION
**Pull request by LinkDragmire**
minor language fixes after playerfeedback

### CODE REVIEW

File: lang/de.json [hunk 1]
```diff
@@ -90,7 +90,7 @@
                 "defenseDifferenceMultiplier": "Der Schaden wird mit der Verteidigungsdifferenz multipliziert (nicht funktionsfähig)",
                 "defenseMultiplierCap": "Der Multiplikator ist begrenzt auf (nicht funktionsfähig)"
             },
-            "DefenseProperties": {
+			"DefenseProperties": {
```
Comment (Stexinator) [inline lang/de.json:93]: ?

File: lang/de.json [hunk 2]
```diff
@@ -100,7 +100,7 @@
             },
             "Settings": {
                 "Alchemical": "Einstellungen für alchemistische Gegenstände",
-                "Armor": "Einstellungen für Rüstung",
+				"Armor": "Einstellungen für Rüstung",
```
Comment (Stexinator) [inline lang/de.json:103]: ?

**Review by Stexinator — CHANGES_REQUESTED** [review]
Looks good but these two weird format mishaps

**Review by LinkDragmire — COMMENTED** [review]
fixed whitespace errors

**Review by Stexinator — APPROVED** [review]

**Pull request merged**